.travis.yml

```diff
@@ -0,0 +1,28 @@
+# Travis CI (https://travis-ci.org/)
```

Comment:
File to add to the repo to let the Travis continued integration service run all the unit tests on the repository: this makes sure that any change we make does not break older stuff